docs/3.0.0/docs/canton/usermanual/pruning.rst

```diff
@@ -43,6 +43,69 @@ For the maximum duration to specify a reliable pruning window "end time", the le
 should should not be wildcards (`*`) as illustrated in the examples above. If the hour field is fixed, so should the
 fields for minute and second.
 
+Schedule Format
+---------------
+
+The format used to describe cron expression consists of 7 whitespace-separated fields:
+
+#. seconds
+#. minutes
+#. hours
+#. day-of-month
+#. month
+#. day-of-week
+#. year (optional)
+
+Each field has a set of valid values:
+
+* seconds and minutes must be a number between ``0`` and ``59``
+
+* hours must be a number between ``0`` and ``23``
+
+* day of the month must be a number between ``1`` and ``31``
+
+* month must be either
+
+  #. a number between ``0`` and ``11`` (with ``0`` being January), or
+
+  #. the first three letters of the month name (``JAN``, ``FEB``, ``MAR``, ...)
+
+* day of the week must be either
+
+  #. a number between ``1`` and ``7`` (with ``1`` being Sunday), or
+
+  #. the first three letters of the weekday name (``SUN``, ``MON``, ``TUE``, ...)
+
+* year must be a number between ``1900`` and ``2099``
```

Comment:
I suggest using a table:

```suggestion
.. list-table::
   :widths: 30 70
   :header-rows: 1

   * - Field
     - Type and valid values\*
   * - seconds
     - number from ``0`` to ``59``
   * - minutes
     - number from ``0`` to ``59``
   * - hours
     - number from ``0`` to ``23``
   * - day of the month
     - number from ``1`` to ``31``
   * - month
     - number from ``0`` to ``11`` (``0`` = January) 
       |br|
       **or** 
       |br|
       the first three letters of the month name (``JAN``, ``FEB``, ``MAR``, ...)
   * - day of the week
     - number from ``1`` to ``7`` (``1`` = Sunday) 
       |br|
       **or** 
       |br|
       the first three letters of the day name (``SUN``, ``MON``, ``TUE``, ...)
   * - year
     - number from ``1900`` to ``2099``

*\*Ranges specified with "to" are inclusive of both endpoints.*
```

It should look like this:
<img width="809" alt="Screenshot 2023-12-03 at 7 01 25 PM" src="https://github.com/digital-asset/docs.daml.com/assets/139921049/a6496846-3832-45c7-affe-6de0d6dda189">

Reply:
If you decide not to keep the table, I'm happy to re-edit to apply the other changes in a list or narrative format. Just let me know. 

See the comment at the top of the page for the line break substitution I used (the table won't work without it).

Reply:
I'm happy to keep the table, but the information that the year is optional went missing in the editing, I'll apply the changes as you suggested them and suggest an edit for you to judge after that. 🙇🏻‍♂️